_episodes/03-index-slice-subset.md

```diff
@@ -16,7 +16,10 @@ objectives:
     - "Locate subsets of data using masks."
     - "Describe BOOLEAN objects in Python and manipulate data using BOOLEANs."
 keypoints:
-    - "FIXME"
+    - "Data can be subset using column headings, numeric ranges , operators, or specific index locations."
```

Comment:
No space before comma in `numeric ranges , `

Reply:
another point: it's not super clear what `Data can be subset` means.

Reply:
```suggestion
    - "Data can be subset using column headings, numeric ranges, operators, or specific index locations."
```

Reply:
Very good point. Now I can't remember why I wanted to add this particular keypoint. Most of the lesson is about using column names to summarize data. Because really, this keypoint summaries this [quick guide to subsetting data in R](https://www.statmethods.net/management/subset.html).

I'm happy to remove it if you think it's irrelevant.

Reply:
space deleted in https://github.com/datacarpentry/python-ecology-lesson/pull/322/commits/034f56aaad5f0332b02d7d7cd1d38238ba8d1fc0

Reply:
This seems to. have bled into the next point, so I'm going to mark this as resolved.

Reply:
My point was that in this lesson we use column headings, ranges, etc for _accessing_ portions of data not for changing it. Because the verb `subset` means `be a subset of`, the phrase `Data can be subset` can be understood as `Data can be changed in-place`

Reply:
Oh, interesting, I had not considered this interpretation. Is there a wording that might be better?

Reply:
Nah :)


<br>
jk! I'm sure there is. Something like


>In Python, portions of data can be accessed using indices, slices, column headings (Panda's data frames), and condition-based subsetting (Panda's data frames and numpy arrays).

Reply:
LGTM

Reply:
This beta suggest and apply feature is awesome!

Reply:
> Does this suggestion work for everyone?

Looks good as long as all of us understand what `condition-based subsetting` is :)

Reply:
```suggestion
    - "In Python, portions of data can be accessed using indices, slices, column headings, and condition-based subsetting."
```

I'll add it in - the beta suggestions thing is a little finnicky to if you changed a prior line.